package.json

```diff
@@ -44,11 +44,14 @@
     "tap-closer": "^1.0.0",
     "tap-status": "^1.0.1",
     "tape": "^4.0.0",
-    "uglify-js": "^2.4.23"
+    "uglify-js": "^2.4.23",
+    "mapbox-gl": "^0.11.0"
+  },
+  "peerDependencies": {
+    "mapbox-gl": "^0.11.0"
```

Comment:
0.12.1 is the latest stable version of mapbox-gl, the peerDependency should include both 0.11.x and 0.12.x since both are API compatible.

You can either use `^0.11.0 || ^0.12.0` to allow both, or take the latest 0.12 and force gl-draw users to upgrade.